src/Comms.h

```diff
@@ -1,6 +1,15 @@
 #pragma once
 
 #include <Arduino.h>
+#include <Ethernet.h>
+#include <EthernetUdp.h>
+#include <map>
+#include <string>
+
+//TODO replace with actual values
+byte mac[] = {0x00, 0x00, 0x00, 0x00, 0x00, 0x00};
+int port = 0;
+IPAddress ip(192, 168, 1, 177);
```

Comment:
The old code had a function to retrieve the Mac address from the teensy. You should find that and use it for this array.

The port should be 42069 and the IP address should be 10.0.0.42